interface/controller/app.go

```diff
@@ -42,31 +42,25 @@ func responseErrorJSON(c Context, err error) error {
 	var status int
 	var r []ErrorResponse
 
-Done:
+BuildResponse:
 	for {
 		switch err.(type) {
 		case nil:
-			break Done
+			break BuildResponse
 		case customErrors.ErrInvalidRequest:
```

Comment:
Pro-tip: you could add more than one value in a case. 
```suggestion
		case customErrors.ErrInvalidRequest, customErrors.ErrDomainValidation:
```